.expeditor/release.omnibus.yml

```diff
@@ -80,8 +81,8 @@ builder-to-testers-map:
     - ubuntu-18.04-x86_64
     - ubuntu-20.04-x86_64
     - ubuntu-22.04-x86_64
-  windows-2012r2-i386:
-    - windows-2012r2-i386
+  # windows-2012r2-i386:
+  #   - windows-2012r2-i386
```

Comment:
Delete instead of comment again, especially since this is not a temporary direction.

Reply:
done